src/components/ImageGeneration/MarkerFiltersDropdown.tsx

```diff
@@ -55,7 +57,7 @@ export function DumbMarkerFiltersDropdown({
   setFilters: (filters: Partial<MarkerFilterSchema>) => void;
 }) {
   const { classes, cx, theme } = useStyles();
-  const mobile = useIsMobile({ breakpoint: 'xs' });
+  const mobile = useMediaQuery('(max-width: 576px)');
```

Comment:
Let's add a comment here explaining why we are using `useMediaQuery` instead of `useIsMobile` or `useContainerQuery`